po/uh-server/unknown-horizons-server.pot

```diff
@@ -1,14 +1,14 @@
 # SOME DESCRIPTIVE TITLE.
 # Copyright (C) YEAR The Unknown Horizons Team
-# This file is distributed under the same license as the Unknown Horizons Server package.
+# This file is distributed under the same license as the PACKAGE package.
```

Comment:
I do not see something where the output change. Maybe my xgettext version (0.19.3) is too old..